run_edx_integration_tests.sh

```diff
@@ -48,8 +50,30 @@ run_plugin_tests() {
 
     # Installing the plugin
     plugin_name=$(basename "$plugin_dir" | sed 's/src\///' | sed 's/_/-/g')
-    tarball=$(ls dist | grep "$plugin_name" | head -n 1)
-    pip install "dist/$tarball"
+
+    # Check if we have a built wheel/tarball for the plugin
+    # Try both underscore and hyphen versions of the name
+    plugin_name_underscore=$(basename "$plugin_dir" | sed 's/src\///')
+
+    tarball=""
+    # Look for wheel first, then tarball
+    for ext in "whl" "tar.gz"; do
+        for name_variant in "$plugin_name" "$plugin_name_underscore"; do
+            found_file=$(ls dist 2>/dev/null | grep -E "^${name_variant}-.*\.${ext}$" | head -n 1)
+            if [ -n "$found_file" ]; then
+                tarball="$found_file"
+                break 2
+            fi
+        done
+    done
+
+    if [ -n "$tarball" ]; then
+        echo "Installing $plugin_name from dist/$tarball"
+        pip install "dist/$tarball"
+    else
+        echo "No built package found for $plugin_name, installing in development mode"
+        pip install -e "$plugin_dir"
+    fi
 
     cp -r /openedx/edx-platform/test_root/ "/openedx/open-edx-plugins/$plugin_dir/test_root"
```

Comment:
Unrelated to your changes but since you're touching this script, the `/openedx/open-edx-plugins/$plugin_dir/test_root` directories need to be gitignored.